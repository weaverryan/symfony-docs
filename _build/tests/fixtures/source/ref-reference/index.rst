A test for reference
====================

:ref:`A ref test <ref-test>`

.. toctree::
    :hidden:

    file
