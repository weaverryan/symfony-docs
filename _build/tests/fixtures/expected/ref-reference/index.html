<!DOCTYPE html>
<html>
<head>
    <meta charset="utf-8" />
</head>
<body>
<div class="section" id="a-test-for-reference">
    <h1>A test for reference<a class="headerlink" href="#a-test-for-reference" title="Permalink to this headline">Â¶</a></h1>
    <p><a href="file.html#ref-test" class="reference internal"><span>A ref test</span></a></p>
</div>
</body>
</html>
